psh/mem generates warning on stm32l4

```
phoenix-rtos-utils/core/psh/mem/mem.c: In function 'psh_mem_procprint':
phoenix-rtos-utils/core/psh/mem/mem.c:85:20: warning: format '%llu' expects argument of type 'long long unsigned int', but argument 3 has type 'id_t' {aka 'unsigned int'} [-Wformat=]
   85 |    printf("  %d.%llu", e->oid.port, e->oid.id);
      |                 ~~~^                ~~~~~~~~~
      |                    |                      |
      |                    long long unsigned int id_t {aka unsigned int}
      |                 %u

```

Example build from ghactions: https://github.com/phoenix-rtos/phoenix-rtos-project/runs/2792844210?check_suite_focus=true#step:4:260

--- a/core/psh/mem/mem.c
+++ b/core/psh/mem/mem.c
@@ -82,7 +82,7 @@ static void psh_mem_procprint(entryinfo_t *e, int mapsz)
 		else if (e->object == OBJECT_MEMORY)
 			printf("  %s", "mem");
 		else
-			printf("  %d.%llu", e->oid.port, e->oid.id);
+			printf("  %d.%llu", e->oid.port, (unsigned long long)e->oid.id);
 
 		if (e->object != OBJECT_ANONYMOUS && e->anonsz != ~0)
 			printf("/(%zuKB)", e->anonsz / 1024);

